pom.xml

```diff
@@ -156,8 +156,8 @@
     <quartz.version>2.2.0</quartz.version>
     <resteasy.version>3.0.8.Final</resteasy.version>
     <rs-api.version>2.0</rs-api.version>
-    <scala2.10.version>2.10.4</scala2.10.version>
-    <scala2.11.version>2.11.8</scala2.11.version>
+    <scala2.10.version>2.10.7</scala2.10.version>
+    <scala2.11.version>2.11.12</scala2.11.version>
```

Comment:
what's the purpose of those version changes ? old version doesn't support Java 11 ?

Reply:
Right. Older Scala version will have an error during the compilation time `scala.reflect.internal.MissingRequirementError: object java.lang.Object in compiler mirror not found`.

Reply:
thanks for the information :)